src/go/rpk/pkg/cli/cmd/redpanda/admin/brokers/brokers.go

```diff
@@ -161,3 +163,64 @@ the cluster leader handles the request.
 		},
 	}
 }
+
+func newDecommissionStatus(fs afero.Fs) *cobra.Command {
+	var completion int
+	return &cobra.Command{
+		Use:   "decommission-status [BROKER ID]",
+		Short: "Show the progress of a node decommissioning",
+		Args:  cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			broker, _ := strconv.Atoi(args[0])
+			p := config.ParamsFromCommand(cmd)
+			cfg, err := p.Load(fs)
+			out.MaybeDie(err, "unable to load config: %v", err)
+
+			cl, err := admin.NewClient(fs, cfg)
+			out.MaybeDie(err, "unable to initialize admin client: %v", err)
+
+			dbs, err := cl.DecommissionBrokerStatus(cmd.Context(), broker)
+			var he *admin.HTTPResponseError
+			if errors.As(err, &he) {
+				body, bodyErr := he.DecodeGenericErrorBody()
+				if bodyErr == nil {
+					out.Die("%s", body.Message)
+				}
+			}
+			if dbs.Finished {
+				out.Die("The node is decommissioned. %d replicas left", dbs.ReplicasLeft)
```

Comment:
Why die here? die exits with 1

also I don't understand the second sentence / what's it mean?

Reply:
Actually I think I understand it -- is there any instance where there will be more than 0 replicas left?

Reply:
> is there any instance where there will be more than 0 replicas left?

In theory, it should be always 0 as long as it's `finished`. I wanted to utilize the raw json output and make it easier to catch a bug in a case non-zero value is returned whereas decommission is marked `finished`.

A raw output when decommissioning is finished.
```
{
  "finished": true,
  "replicas_left": 0
}
```
Do you think if's too much?

Reply:
I'd lean toward the following way:
If `finished` is true and
* if `replicas_left == 0`, print `The node is decommissioned successfully`
* else, print `The node is decommissioned but xx replicas left`

Wdyt?

Reply:
What should a user do if xx replicas are left?

Reply:
That's so far unknown, tbh. It's most likely a bug in Redpanda if that happens.

Reply:
Should we encourage the users to report to us in GH if they hit this?
```
The node is decommissioned but %d replicas left. There is likely an issue inside Redpanda. Please describe how you encountered this at https://github.com/redpanda-data/redpanda/issues/new?assignees=&labels=kind%2Fbug&template=01_bug_report.md
```

Reply:
@mmaslankaprv , what do you think is a better advise in a case replicas_left is non-zero whereas decommission is marked finished? Shouldn't we consider about that kind of abnormal case in general?

Reply:
Nvm, I've removed the check in a case `replicas_left` is non-zero since it's must be a quite rare case.

Reply:
We have one other area inside rpk where we print "if you run into this, please file a bug" -- so I'm open to that, too, fwiw.

Reply:
Yea, I've found it so that come with the idea. I've reverted the check for better supportability. Thanks!